tools/nut-scanner/scan_xml_http.c

```diff
@@ -62,7 +62,6 @@ static nutscan_device_t * dev_ret = NULL;
 #ifdef HAVE_PTHREAD
 static pthread_mutex_t dev_mutex;
 #endif
-long g_usec_timeout ;
```

Comment:
@aquette : was there any use or plan for this variable, or was it just some copy-paste artifact?